### ISSUE TITLE
Archive this repository?

### ISSUE THREAD

**Issue by BendingBender** [sindresorhus/number-is-integer#1]
With the advent of Node.js 8 as the oldest LTS version, this is actually not needed any more and should be archived.

**Comment by sindresorhus** [comment]
Yeah, I have a lot of these polyfill modules. But there are still a lot of people using it, so I don't really see any harm in just leaving it as is for now. I plan to npm deprecate and archive them at some point, but don't think it's worth it to inconvenience people yet. You can just ignore adding TS definitions to these polyfill modules.
